lib/docassemble/takedown.js

```diff
@@ -10,9 +10,15 @@ const takedown = async () => {
   *   Project created for this test.
   *   For local development, clean up project name file. */
   log.info({ type: `takedown`, pre: `Trying to remove the docassemble Project from the testing account.` });
-  await da_i.delete_project();
-  session_vars.delete_project_name();
-  log.info({ type: `takedown`, pre: `Success! Test takedown has finished successfully.` });
+  try {
+    await da_i.delete_project();
+    session_vars.delete_project_name();
+    log.info({ type: `takedown`, pre: `Success! Test takedown has finished successfully.` });
+    process.exitCode = 0;
```

Comment:
I'm still not quite sure what the ramifications would be downstream (whatever github or our action script is doing). It seems safe, but why not let it complete on its own?

Answer:
1. Setting exit code doesn't stop the process from running, just affects the code.
2. It's implicitly always 0.

Response:
It still seems a bit weird to do it here. If in future we actually handle unhandled promise rejection, we'll be removing these exit code things anyway, so we'll just have to then also remember to clean this up. This `process.exitCode` stuff is, from what I understand, a hack to avoid the useless error. This isn't a new practice for our code in general.

Are we trying to follow some kind of best practices?

Answer: We may end up using `process.exitCode` in the handler

Response: Possibly true, but we won't be using it in takedown.js, etc. When we get rid of line 20, would we get rid of line 17? To which we think the answer is 'yes'. So line 20 is just an intermediate step and we might not want to include line 17 as part of that intermediate step.

Answer: Just thought it added clarity in this specific spot.

Response: With that justification, I'm ok either way.

Reply:
I'd possibly want a comment on why we're using it here too because it's an unusual thing to see.